web/app/controller/Root.js

```diff
@@ -25,6 +25,10 @@ Ext.define('Traccar.controller.Root', {
         'Traccar.model.Position'
     ],
 
+    init: function () {
+        Ext.state.Manager.setProvider(new Ext.state.CookieProvider());
```

Comment:
This is amazing. Can it save status of everything else as well (e.g. buttons on the map)? This way we won't need to save them in the user attributes.